lib/health_checks/sidekiq.rb

```diff
@@ -36,16 +40,29 @@ def start
       end
     end
 
-    def check_redis
-      sidekiq_response = ::Sidekiq.redis(&:ping)
-      return if sidekiq_response == 'PONG'
+    def create_clients(mongo_databases)
+      mongo_databases.map do |db|
+        client_name = "#{db[:name]}_sidekiq_health_check"
+        client = MongoidCustomClient.create(client_name, db[:hosts])
+
+        { db_name: db[:name], instance: client }
+      end
+    end
 
-      raise "Sidekiq.redis.ping returned #{sidekiq_response.inspect} instead of PONG"
+    def check_mongoid_clients(clients)
+      db_name = ''
+      clients.each do |client|
+        db_name = client[:db_name]
+        client[:instance].command(dbStats: 1).first['db']
+      end
+    rescue => e
```

Comment:
Only rescue the piece of code that you expect to raise. It will also make the code simpler in this case, no need for the `db_name = ''`

Reply:
yeah, it is a hackish way of printing out the db name that failed. Any suggestion how to do it in a proper way?

Reply:
???

Reply:
???

Reply:
¿¿¿